udapi/block/read/conllu.py

```diff
@@ -86,6 +105,19 @@ def process_document(self, document):
                 logging.debug('No next bundle to process')
                 break
 
+            # Check if all lines have correct number of data fields.
+            # Skip invalid bundle otherwise.
+            raw_bundle_check = True
+            for line in raw_bundle:
+                if re_comment_like.search(line) is not None:
+                    continue
+                if len(line.split('\t')) != len(self.node_attributes):
```

Comment:
Should I add a special parameter `strict` which will enable more controlled processing?